apps/aeutils/test/data/epoch_full.yaml

```diff
@@ -138,8 +138,12 @@ chain:
     db_path: .
     # Hard forks
     hard_forks:
-      "1": 0
-      "2": 1
+      "1":
+        height: 0
+        use_hardcoded_directory : true
+      "2":
+        height: 1
+        directory: "/tmp/1"
```

Comment:
Does it also allow relative to the node root path?

Reply:
No I can add that.